bin/wmagent-couchapp-init

```diff
@@ -5,17 +5,16 @@ wmagent-couchapp-init
 """
 from __future__ import print_function, division
 
+from future import standard_library
+standard_library.install_aliases()
+
 import argparse
 import os
 import shutil
 import sys
 import tempfile
 import urllib
 
-try:
-    from urlparse import urlsplit
-except ImportError:
-    # PY3
     from urllib.parse import urlsplit
```

Comment:
indentation

Reply:
Oops, my bad. Having a second pair of eyes going through the changes is always useful!